valgrind.supp

```diff
@@ -0,0 +1,965 @@
+   fun:_ZNK5boost7phoenix9evaluator4implIRNS0_5actorINS_5proto7exprns_10basic_exprINS4_6tagns_3tag10shift_leftENS4_7argsns_5list2INS3_INS6_IS9_NSB_INS3_INS_3log11v2_mt_posix11expressions3aux25attribute_output_terminalINS3_INS6_IS9_NSB_INS3_INSG_INS3_INS6_IS9_NSB_INS3_INSG_INS3_INS6_IS9_NSB_INS3_INS6_IS9_NSB_INS3_INS6_IS9_NSB_INS3_INSG_INS3_INS6_INS8_8terminalENSA_4termINS0_8argumentILi2EEEEELl0EEEEENS_10posix_time5ptimeENSD_16fallback_to_noneENSD_3aux14light_functionIFvRNSD_24basic_formatting_ostreamIcNSt3__111char_traitsIcEENSU_9allocatorIcEEEERKSP_EEEEEEENS3_INS6_ISH_NSI_IA7_cEELl0EEEEEEELl2EEEEENS3_INS6_ISH_NSI_IiEELl0EEEEEEELl2EEEEENS3_INS6_ISH_NSI_IA3_cEELl0EEEEEEELl2EEEEEN2ml4core7CLogger6ELevelESQ_NSD_10to_log_funIvEEEEEENS3_INS6_ISH_NSI_IcEELl0EEEEEEELl2EEEEENSU_12basic_stringIcSW_SY_EESQ_S1W_EEEES21_EELl2EEEEEiSQ_S1W_EEEES21_EELl2EEEEENSE_15attribute_actorIS26_SQ_NSE_3tag8smessageES3_EEEELl2EEEEERKNS0_7vector2IRNS0_7vector3IPKS2N_RKNSD_11record_viewERNSF_10stream_refISZ_EEEERKNS0_15default_actionsEEENS4_6envns_9empty_envEEclES2O_S36_RKS38_
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Value8
+   fun:_platform_memmove$VARIANT$Haswell
+   fun:__sfvwrite
+   fun:__vfprintf
+   fun:__v2printf
+   fun:_vsnprintf
+   fun:vsnprintf_l
+   fun:snprintf_l
+   fun:_ZNKSt3__17num_putIcNS_19ostreambuf_iteratorIcNS_11char_traitsIcEEEEE6do_putES4_RNS_8ios_baseEcd
+   fun:_ZNSt3__113basic_ostreamIcNS_11char_traitsIcEEElsEd
+   fun:_ZNSt3__1lsIdcNS_11char_traitsIcEEEERNS_13basic_ostreamIT0_T1_EES7_RKNS_7complexIT_EE
+   fun:_ZN2ml4core14printer_detail12CLeafPrinterIN5boost17integral_constantIbLb0EEEE6print_INSt3__17complexIdEEEENS8_12basic_stringIcNS8_11char_traitsIcEENS8_9allocatorIcEEEERKT_S5_
+   fun:_ZN2ml4core14printer_detail12CLeafPrinterIN5boost17integral_constantIbLb0EEEE5printINSt3__17complexIdEEEENS8_12basic_stringIcNS8_11char_traitsIcEENS8_9allocatorIcEEEERKT_
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Cond
+   fun:_ZNSt3__115basic_stringbufIcNS_11char_traitsIcEENS_9allocatorIcEEE8overflowEi
+   fun:_ZNSt3__115basic_streambufIcNS_11char_traitsIcEEE6xsputnEPKcl
+   fun:_ZNSt3__115basic_streambufIcNS_11char_traitsIcEEE5sputnEPKcl
+   fun:_ZNSt3__116__pad_and_outputIcNS_11char_traitsIcEEEENS_19ostreambuf_iteratorIT_T0_EES6_PKS4_S8_S8_RNS_8ios_baseES4_
+   fun:_ZNSt3__124__put_character_sequenceIcNS_11char_traitsIcEEEERNS_13basic_ostreamIT_T0_EES7_PKS4_m
+   fun:_ZNSt3__1lsIcNS_11char_traitsIcEENS_9allocatorIcEEEERNS_13basic_ostreamIT_T0_EES9_RKNS_12basic_stringIS6_S7_T1_EE
```

Comment:
Similarly this stack trace could end at `_ZNSt3__1lsIcNS_11char_traitsIcEENS_9allocatorIcEEEERNS_13basic_ostreamIT_T0_EES9_RKNS_12basic_stringIS6_S7_T1_EE` and cover many similar ones.